components/automate-cli/pkg/docs/constants.go

```diff
@@ -11,4 +11,5 @@ const (
 	PostgreSQL               = "Postgresql"
 	Automate                 = "Automate"
 	FrontEnd                 = "FrontEnd"
+	AutomateConfigTimeout    = int64(10)
```

Comment:
Let's not keep the code constants here. This is specifically for docs generation